src/main/resources/log4j.properties.EXAMPLE

```diff
@@ -0,0 +1,10 @@
+log4j.rootLogger=INFO, stdout, R
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
```

Comment:
I think we can basically. just copy over the one that we have the CVR/cmo-pipielines proj